code/modules/power/cable.dm

```diff
@@ -514,7 +514,7 @@ GLOBAL_LIST(cable_radial_layer_list)
 			span_warning("You patch some wires in \the [H]'s [affecting.display_name]."))
 		if(affecting.heal_limb_damage(0, 15, robo_repair = TRUE, updating_health = TRUE))
 			H.UpdateDamageIcon()
-		if(!amount)
+		if(amount < 1) // is this check really needed since we check use()
```

Comment:
Because use is in the while statement, does that means when this if statement is checked, amount could be zero if we used the last coil? Or does it function differently?

Reply:
Use will qdel it on the last use then it will return here, and if I remove it it will go for another loop before trying to use
I cant be bothered to change it here so I just removed the comment